powa--5.1.1.sql

```diff
@@ -346,7 +346,8 @@ INSERT INTO @extschema@.powa_db_module_src_queries
         heap_blks_read, heap_blks_hit, idx_blks_read, idx_blks_hit,
         toast_blks_read, toast_blks_hit, tidx_blks_read, tidx_blks_hit
      FROM pg_catalog.pg_stat_all_tables st
-     JOIN pg_catalog.pg_statio_all_tables sit USING (relid)'),
+     JOIN pg_catalog.pg_statio_all_tables sit USING (relid)
+     WHERE st.schemaname <> ''pg_toast'''),
```

Comment:
should we use `NOT LIKE 'pg_toast%'` instead?

as far as I can see temp tables with toastable attribute ends up with their toast in scheme named `pg_toast_temp_XXX`

for instance
```sql
=# create temp table blabla(val text);
CREATE TABLE

=# select reltoastrelid::regclass from pg_class where relname = 'blabla';
         reltoastrelid          
--------------------------------
 -
 pg_toast_temp_4.pg_toast_19711
(2 rows)
```

Reply:
Yeah, I didn't think about temp tables…